tfx/examples/imdb/imdb_utils_native_keras.py

```diff
@@ -0,0 +1,213 @@
+# Lint as: python2, python3
```

Comment:
please double check the diff between this file and iris_utils_native_keras.py, other than the tuner part and some function contents (), structure should be about the same

Reply:
yes